packages/@aws-cdk/aws-apigatewayv2/test/http/api.test.ts

```diff
@@ -265,12 +264,12 @@ describe('HttpApi', () => {
     api.addVpcLink({ vpc, vpcLinkName: 'Link-2' });
 
     // THEN
-    expect(stack).toHaveResource('AWS::ApiGatewayV2::VpcLink', {
+    Template.fromStack(stack).hasResourceProperties('AWS::ApiGatewayV2::VpcLink', {
       Name: 'Link-1',
     });
-    expect(stack).not.toHaveResource('AWS::ApiGatewayV2::VpcLink', {
+    expect(Template.fromStack(stack).findResources('AWS::ApiGatewayV2::VpcLink', {
       Name: 'Link-2',
-    });
+    }).length).toEqual(0);
```

Comment:
This feels a little clunky...is there a plan to add resource properties to `resourceCountIs`? Or for supporting a `not` operator that catches assertion errors/throws if nothing was thrown?

Reply:
Opened a feature request - https://github.com/aws/aws-cdk/issues/15868